utilities/EluvioLiveCli.js

```diff
@@ -2754,6 +2756,14 @@ yargs(hideBin(process.argv))
           describe: "Replace the transfer proxy address (hex). The proxy owner must be the owner of this contract.",
           type: "string",
         })
+        .option("mint_shuffle_key", {
```

Comment:
consider putting the command line examples in https://github.com/qluvio/elv-master/pull/935#issue-1656546896

in the README markdown here for elv-live-js so we have good example of mint_shuffle_key and legacy_shuffle_seed here.

Reply:
Done!